ec2-cluster/modules/node/main.tf

```diff
@@ -33,7 +33,7 @@ resource "aws_instance" "node" {
 
   tags = {
     Index     = count.index
-    Name      = "node-${var.cluster_id}-${count.index}"
+    Name      = "node-${var.env}-${var.cluster_id}-${count.index}"
```

Comment:
I think the shorter the better, having all tags available to display.
![image](https://github.com/user-attachments/assets/ae576503-1363-43b3-8112-9d1bbab2e3ac)

Reply:
👍🏼